testData/deviceReporter/installedSolutions.json

```diff
@@ -113,6 +113,10 @@
 
     {
         "id": "com.android.settings.system"
+    },
+
+    {
+        "id": "net.gpii.browserExtension"
```

Comment:
From my conversation with @kaspermarkus in the [fluid-work channel](https://botbot.me/freenode/fluid-work/2017-08-16/?msg=89914220&page=1), this file should not be modified here. However, the UIO+ solution will not work. To make it work, the file should be manually modified to include the solution ID.